torch/optim/_multi_tensor/rmsprop.py

```diff
@@ -78,7 +102,7 @@ def step(self, closure=None):
                         raise RuntimeError('RMSprop does not support sparse gradients')
 
                     grads.append(p.grad)
-                    params_with_grad.append(p)
+                    params_with_grad.append(p.data)
```

Comment:
We recently cleaned up the use of `.data` inside optimizers. Are you aware of its interaction with multitensor?